models/topic_models.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../db/connection');
+
+exports.fetchTopics = () => {
+    return db
+    .query(`SELECT * FROM topics`)
+    .then(({rows}) => {
+        if(rows.length === 0){
+            return Promise.reject({status:404, msg:"No Topics Found"})
+        }
+        return rows
+    });
```

Comment:
This is fine but not strictly needed here as we can assume that asking for the topics, however many there are, won't return an error. This is more for when a user asks for a specific thing like a single topic, article or comment which doesn't exist.

Reply:
Overall great work though 👍